### PR TITLE
add google analytics tracking to newlibs, propagate ga_code to JS config

### PR DESCRIPTION
**Pull request by martenson**

### CODE REVIEW

**Comment by martenson** [comment]
@galaxybot test this

**Comment by martenson** [comment]
asking for a review

File: client/galaxy/scripts/galaxy.library.js [hunk 1]
```diff
@@ -3,43 +3,49 @@
 // MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
```
Comment (carlfeberhard) [inline client/galaxy/scripts/galaxy.library.js:3]: image

**Comment by carlfeberhard** [comment]
Looks good!

**Pull request merged**